allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/results/AllureResultInjector.kt

```diff
@@ -0,0 +1,44 @@
+package com.kaspersky.components.alluresupport.results
+
+import androidx.test.uiautomator.UiDevice
+import java.io.File
+
+class AllureResultInjector(
```

Comment:
A separate entity responsible to replace all stubs in the results dir with real videos